.travis.yml

```diff
@@ -157,11 +157,11 @@ jobs:
 
     # === integration with current platform
     - stage: ❧ pdb tests
-      env: PDB_TEST=int/openjdk10/pup-6.0.x/srv-6.0.x/pg-9.6
+      env: PDB_TEST=int/openjdk11/pup-6.0.x/srv-6.0.x/pg-9.6
```

Comment:
I'm a little surprised that this worked, it has failed for me before and the change that supposed jdk11 for puppetserver first hit `6.1.0` https://github.com/puppetlabs/puppetserver/commit/327a1ae23d4c7f9b0a2a3a080bd55ebc26855499